openbb_platform/extensions/equity/openbb_equity/discovery/discovery_router.py

```diff
@@ -11,7 +11,7 @@
 from openbb_core.app.router import Router
 from pydantic import BaseModel
 
-router = Router(prefix="/disc")
+router = Router(prefix="/discovery")
```

Comment:
`equity_discovery_undervalued_growth_equities` could trim`equities` from the end.

Reply:
![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/11668535/a30ace38-9e43-42a3-9498-76232ad2a080)